Jenkinsfile

```diff
@@ -66,6 +66,7 @@ def buildImageAndRunTests(params, String suffix) {
         if (GlobalVars.isTestSuiteJob) {
             testExtraArgs += ['--kyua-tests-files', '/usr/tests/Kyuafile',
                               '--no-run-cheribsdtest', // only run kyua tests
+			      '--disable-coredumps',  // coredumps slow down the testsuite unnecessarily
```

Comment:
```suggestion
                              '--disable-coredumps',  // coredumps slow down the testsuite unnecessarily
```

Reply:
It looked aligned in the web UI :(

Reply:
Ah, the web editor was inserting tabs instead of spaces.